doc/guidelines/development/docker/index.rst

```diff
@@ -2,10 +2,113 @@
+
+When you open the Visual Studio Code and go to the docker icon (whale shape) on the left column, you will see the list of images installed and containers you opened.  
+
+.. figure:: ../../../images/visual_studio_docker.png
+    :width: 400px
+
+Right click while pointing on the container you want to use, and select "attach Visual Studio Code". It will open your container into the new window.
+
+In the local terminal where you just ran the command to open the container, or from the terminal in the visual studio code, run ::
+    
+    bash ~/.setup_dev.sh
+
+.. image:: ../../../images/setup_dev.png
+  :width: 700px
+
+Then go to your repo, which you put into the "--volume", ::
+    
+    cd /home/saluser/develop/(path_to_repos)/ts_standardscripts
+
+then run following commands::
+    
+    setup -r .
+    eups list
+
+You should see something like this::
+
+    ...
+    ts_scriptqueue        tag:current       current setup
+    ts_simactuators       tag:current       current setup
+    ts_standardscripts    LOCAL:/home/saluser/develop/lsst-ts/ts_standardscripts    setup
+    ts_standardscripts    tag:current       current
+    ts_tcpip              tag:current       current setup
+    ... 
+
+This means that you are using your version of the ts_standardscripts instead of the one that comes with the docker image.
+
+In visual studio code, you can also switch branch from the bottom of the window as shown in the image below, click the branch icon, and all the branches will appear on the top box where it was the search box previously.
+
+.. image:: ../../../images/branch_vsc.png
+  :width: 700px
+
+You will NOT see the branches that are up-to-date in git hub because the container is made from the image, and the image is not necesarrily following up with the git hub repository. 
+This is where the repo you called from "--volume" is needed. 
+
+The container default repository is::
+
+    /home/saluser/repos/
+
+while repo is under as you declared in "--volume"::
+
+    /home/saluser/develop/
+
+Now your environment is ready. You can open and edit the script from clicking the file on the left pannel. You can run the files under the /home/saluser/develop/ in terminal (bottom red box in the following image). For example, if you run test code, you can quickly check if your code will pass or fail under the environment.
+
+.. image:: ../../../images/open_from_repo.png
+  :width: 700px
+
+If you want to work with github branch in your container, you can clone the git repository as well, clicking *Clone Git Repository* and put the URL of the git repository you want to clone.
+
+.. image:: ../../../images/branch_git_repo.png
+  :width: 700px
+  
+Remember, once you close and delete the container, changes made in the container is not saved nor updating original image files. Therefore, changes should be copied and saved in local repo (--volume). 
```

Comment:
Shouldn't use the plural here?

changes made in the container **is** not saved > changes made in the container **are** not saved